python2-pybind11/setup.py

```diff
@@ -3,7 +3,7 @@
 from distutils.core import setup, Extension
 from distutils import sysconfig
 
-cpp_args = ['-std=c++14', '-stdlib=libc++', '-mmacosx-version-min=10.7']
```

Comment:
Yes, I'd already stolen the bit that detects the pybind11 location using pip from that example; I didn't think the extra logic in the setup.py was worth it just for C++14, especially when most consumers of this package will be looking at the code, not the binary.  But I'll add a comment to that effect.

It doesn't look like `-fvisibility-hidden` is going to be worth the effort What doesn't work here is that I'm actually packaging the C++ library object files into the Python module (and hence using Python's dynamic linking as the only linking that needs to happen).  That simplifies the build, but coupled with distutils refusal to let me compile object files within the same extension differently, it means I'd have to add a lot of visibility attributes to the actual library code.  That may be good practice, but it'll confuse my intended audience (other people in our project, which doesn't have a practice of adding visibility attributes), so for now I'll just add a comment for that, too.